taskboard/taskboard.sh

```diff
@@ -278,7 +285,8 @@ while :
 do
 	menu "\
 Q: Quit TaskBoard | N: New Task       | X: Close Selected
-[Enter]: Activate/Deactivate Selected | M: More Options" "$(load_items "$ITEMS_DIR")" $selected 'Q' 'N' 'X' 'M'
+[Enter]: Activate/Deactivate Selected | M: More Options
+O: Open Associated Task in JIRA       | " "$(load_items "$ITEMS_DIR")" $selected 'Q' 'N' 'X' 'M' 'O'
```

Comment:
Could this use "J" instead of "O"? I might want to reserve "O" for a more general opening task, if I ever need to add one. Also could this text be on the middle line in the GUI instead of the bottom line? I want to keep "[Enter]: ..." on the bottom.